andela/AndelaEatsUI
# #161079281 Implement edit engagement functionality

#### What does this PR do?
- Implements `Edit Engagements` functionality.

#### Description of Task to be completed?
- Ensures an admin can edit an engagement

#### How should this be manually tested?
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git) and
-  cd into AndelaEatsUI and checkout to this branch `ft-admin-edit-engagement-functionality-161079281`
- run `npm run bundle` and `npm run dev`
- Log in as an `Admin`, navigate to `admin dashboard`, select `Engagements`, and click on `Edit`
- run `npm test` to ensure all tests are passing


#### What are the relevant pivotal tracker stories?
- [#161079281](https://www.pivotaltracker.com/n/projects/2178749/stories/161079281)

#### Background Context
Nil

#### Screenshots (If Applicable)
<img width="1278" alt="screen shot 2018-11-15 at 9 03 51 pm" src="https://user-images.githubusercontent.com/21138774/48607539-b023f680-e981-11e8-9c6f-11a43b969dc7.png">
